pose_frame_semantics/tutorial.md

```diff
@@ -1,23 +1,82 @@
 # Pose Frame Semantics Tutorial
 
 This documentation explains the implemented changes proposed by
-[Pose Frame Semantics Proposal](http://sdformat.org/tutorials?tut=pose_frame_semantics_proposal) in SDFormat 1.7.
+[Pose Frame Semantics Proposal](http://sdformat.org/tutorials?tut=pose_frame_semantics_proposal) in SDFormat 1.7,
+using visuals from the [Pose Frame Specification for SDFormat 1.7](https://doi.org/10.36288/ROSCon2019-900911)
+presentation at [ROSCon 2019](https://roscon.ros.org/2019/).
 
 Prerequisites:
 
 - [Specifying Pose in SDFormat](http://sdformat.org/tutorials?tut=specify_pose&cat=specification&#specifying-pose-in-sdformat)
 
 - [Specifying model kinematics in SDFormat](http://sdformat.org/tutorials?tut=spec_model_kinematics&cat=specification&#specifying-model-kinematics-in-sdformat)
 
+## Syntax
+
+This document uses [XPath syntax](https://www.w3schools.com/xml/xpath_syntax.asp)
+to describe elements and attributes concisely.
+For example, `<model>` tags are referred to as `//model` using XPath.
+XPath is even more concise for referring to nested tags and attributes.
+In the following example, `<link>` elements inside `<model>` tags are
+referenced as `//model/link` and  model `name` attributes as `//model/@name`:
+
+    <model name="model_name">
+      <link/>
+    </model>
+
 ## What's New In SDFormat 1.7 With Examples
 
+The primary new features in SDFormat 1.7 are specifying poses relative to a
+frame of your choice using `//pose/@relative_to` and creating named frames
+using `//frame`. A model consisting of two links (`base` and `pendulum`)
+and one revolute joint is used to illustrate these new features.
+
 [[file:pendulum.png|400px]]
 
-[[file:pendulum_demo.gif|465px]]
+### Specifying relative poses with `//pose/@relative_to`
 
-### `//pose/@relative_to`
+SDFormat 1.6 and earlier have
+[fixed rules for each type of element](/tutorials?tut=pose_frame_semantics&ver=1.5#parent-frames-in-sdf-1-4)
+to determine the frame in which a pose is specified. For example, all link
+poses are specified relative to the model frame. This is illustrated for the
+pendulum with base model in SDFormat 1.6 format and an illustration.
 
-A new `/@relative_to` attribute for `//pose` tags to specify the relative relation of the current pose to another entity pose.
+~~~
+<sdf version="1.6">
+  <model name="pendulum_with_base">
+    <link name="base">
+      <pose>0 0 0.3   0 0 0</pose>
+    </link>
+    <link name="pendulum">
+      <pose>0 0.5 1.0   1.57 0 0</pose>
+    </link>
+    <joint name="joint" type="revolute">
+      <parent>base</parent>
+      <child>pendulum</child>
+      <pose>0 0 0.5   0 0 0</pose>
+      <axis>
+        <xyz>1 0 0</xyz>
+      </axis>
+    </joint>
+  </model>
+</sdf>
+~~~
+
+[[file:pendulum_16.png|400px]]
+
+With SDFormat 1.7, a pose value can be specified relative to a another frame
```

Comment:
nit: delete `a` 
 relative to a another frame ->  relative to another frame

Reply:
https://github.com/gazebosim/sdf_tutorials/pull/83/commits/1fb415938d8a5ffe35de480926ca214ba7029ac5